Makefile

```diff
@@ -188,6 +209,8 @@ clean:
 	-docker rmi -f $(DOCKER_HUB_REPO)/torpedo-pds:$(DOCKER_HUB_TAG)
 	@echo "Deleting backup image"
 	-docker rmi -f $(DOCKER_HUB_REPO)/torpedo-backup:$(DOCKER_HUB_TAG)
+	@echo "Deleting gin image"
```

Comment:
You can call it `Deleting torpedo-api image`

Reply:
Updated the message to "Deleting taas image"